presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -1530,4 +1531,16 @@ public FeaturesConfig setEnforceFixedDistributionForOutputOperator(boolean enfor
         this.enforceFixedDistributionForOutputOperator = enforceFixedDistributionForOutputOperator;
         return this;
     }
+
+    public boolean isEmptyJoinOptimization()
+    {
+        return optimizeJoinsWithEmptySources;
+    }
+
+    @Config("optimizer.is-empty-join-optimization")
```

Comment:
nit: This should be "optimizer.optimize-join-with-empty-sources"

Reply:
OK